blocks/api/serializer.js

```diff
@@ -93,9 +93,11 @@ export function getCommentAttributes( allAttributes, attributesFromContent ) {
 			const contentValue = attributesFromContent[ key ];
 
 			// save only if attribute if not inferred from the content and if valued
-			return ! ( contentValue !== undefined || allValue === undefined )
-				? Object.assign( toSave, { [ key ]: allValue } )
-				: toSave;
+			if ( ! isNil( contentValue ) || isNil( allValue ) ) {
+				return toSave;
+			}
+
+			return Object.assign( toSave, { [ key ]: allValue } );
```

Comment:
It would be nice to break this into two PRs: one that modifies the behavior of the left alignment; one that changes what it means to save attributes.

To this point I haven't heard discussion about whether we should save `null` values and while I could see value in trimming them out, I think it's worth making sure we don't want to save them.

In any case, was there a reason to not only change the condition but also change the code structure?